circus-train-integration-tests/src/test/java/com/hotels/bdp/circustrain/integration/utils/ThriftMetastoreServerRuleExtension.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (C) 2016-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.integration.utils;
+
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.hive.conf.HiveConf.ConfVars;
+
+import com.hotels.beeju.ThriftHiveMetaStoreJUnitRule;
+
+public class ThriftMetastoreServerRuleExtension extends ThriftHiveMetaStoreJUnitRule {
```

Comment:
Is this class just a convenience to override the conf values?

Reply:
Pretty much yep, `beforeTest()` is called from within beeju, so this is where we will initialise beeju with the hiverunner metastore

Reply:
OK

Reply:
We should push this to Beeju once and fix it there (We also do this in other internal projects).

Reply:
Agreed, can someone do that now? It not can we add an issue over at BeeJU and refer to this?